x/incentives/keeper/utils_test.go

```diff
@@ -156,16 +155,7 @@ func (suite *KeeperTestSuite) Commit() {
 
 // Commit commits a block at a given time.
 func (suite *KeeperTestSuite) CommitAfter(t time.Duration) {
-	_ = suite.app.Commit()
-	header := suite.ctx.BlockHeader()
-	header.Height++
-	header.Time = header.Time.Add(t)
-	suite.app.BeginBlock(abci.RequestBeginBlock{
-		Header: header,
-	})
-
-	// update ctx
-	suite.ctx = suite.app.BaseApp.NewContext(false, header)
+	suite.ctx = testutil.Commit(suite.ctx, suite.app, t)
 
 	queryHelper := baseapp.NewQueryServerTestHelper(suite.ctx, suite.app.InterfaceRegistry())
```

Comment:
why are we updating the query clients during commit?

Reply:
This is a pattern that repeats on all test suites. The query client is updated with the new context. Tried removing this but tests fail